x-pack/plugins/visualization_lens/public/types.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface EditorFrameAPI {
```

Comment:
nit: The type naming convention that Core has been using for setup return types would be `EditorFrameSetup`